conans/client/rest/version_checker.py

```diff
@@ -69,8 +69,3 @@ def _handle_ret(self, ret):
                           "Please, contact with your system administrator" \
                           " and upgrade the server." % server_version
                     raise ConanOutdatedClient(msg)
-                else:
-                    msg = "The conan remote version is outdated (v%s). " \
```

Comment:
No longer telling about the remote being outdated?

Reply:
I think that unless we can identify the conan_server we should not warn. Because Artifaactory and Bintray are always outdated, but they are ok. It was only a debug trace indeed.